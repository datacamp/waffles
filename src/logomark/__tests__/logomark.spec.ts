import * as logomarks from '../index';

describe('Logomark', () => {
  it('exports every logomarks', () => {
    expect(logomarks).toMatchSnapshot();
  });
});
