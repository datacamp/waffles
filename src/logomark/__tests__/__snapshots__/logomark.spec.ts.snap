// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logomark exports every logomarks 1`] = `
Object {
  "ExcelLogomark": [Function],
  "GitLogomark": [Function],
  "PowerBiLogomark": [Function],
  "PythonLogomark": [Function],
  "RLogomark": [Function],
  "ScalaLogomark": [Function],
  "ShellLogomark": [Function],
  "SpreadsheetLogomark": [Function],
  "SqlLogomark": [Function],
  "TableauLogomark": [Function],
  "TheoryLogomark": [Function],
}
`;
